public/app/common/components/cc-form/ccTextInput.jsx

```diff
@@ -0,0 +1,40 @@
+require('react');
+var cx = require('react/lib/cx');
+
+function ccFormInput (locals) {
+  var formGroupClasses = {
+    'cc-form-item': true,
+    'cc-form-item--error': locals.hasError
+  };
+
+  var inputClasses = {
+    'cc-form-item__input': true,
+    'cc-form-item__input--error': locals.error
+  };
+
+  return (
+    <div className={cx(formGroupClasses)}>
+      {
+        locals.label ?
```

Comment:
When wrapping with ternary operator, align code this way:

    expression
        ? this
        : or that